tasks/debian.yml

```diff
@@ -1,6 +1,14 @@
 ---
-- name: debian | Installing DRBD
+- name: debian | Installing DRBD packages
   apt:
-    name: drbd8-utils
+    name: "{{ item }}"
     state: present
   become: true
+  with_items: "{{ drbd_additional_packages }}"
```

Comment:
We could change this to `drbd_debian_additional_packages` for Debian based apt packages. This would allow the flexibility to not have to define this variable differently between Debian and RHEL based OS's.